engines/billing/questions.md

```diff
@@ -0,0 +1,22 @@
+Questions
+====================
+
+1. Should admin be able to see subscriptions?
+
+2. Should user be able to upgrade or downgrade subscriptions?
+
+3. Should confirmations emails be allowed?
+
+See here:
+https://articles.braintreepayments.com/guides/recurring-billing/email-notifications
+https://articles.braintreepayments.com/control-panel/transactions/email-receipts
+
+4. Braintree uses JS for dropin. I'm not sure but this may not work in browsers
+such as Tor. Is there a problem?
+JS can be improved and changed to 'Hosted Fields'. See here:
+https://developers.braintreepayments.com/javascript+ruby/guides/hosted-fields/overview
```

Comment:
I would say we can look into this if the problem comes up. I don't think there is much benefit of connecting to the provider via tor - at least not if you are using the providers vpn anyway. I think there are other more important issues that would also reduce our dependency on js such as allowing login from the client to access the webapp.